spacelift/data_stack.go

```diff
@@ -154,6 +154,20 @@ func dataStack() *schema.Resource {
 				Description: "Name of the Docker image used to process Runs",
 				Computed:    true,
 			},
+			"showcases": {
+				Type:        schema.TypeList,
+				Description: "Showcase-related attributes",
+				Computed:    true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"namespace": {
```

Comment:
What does `namespace` stand here for?

Reply:
Now it will always be spacelift-io. In the future we might want to have showcases in a different org, so I left this be.

Reply:
Wouldn't it be like an ID of a showcase? Like, you could have a showcase with a Minecraft server or something similar?

Reply:
Yeah, but I assume showcases are stored in GitHub, and we just put namespace+repo names into an allow list.

Reply:
I get it now, thanks.